README.md

```diff
@@ -224,30 +224,16 @@ serializer.save()
 The `EnumChoiceModelSerializerMixin` does not need to be used if `enumerated_field` is defined on the serializer class explicitly.
 
 
-## Customizing readable values
```

Comment:
Lets leave this but change the examples with the different choice builders + provide an example for how to do this on your own